llvm/test/tools/sycl-post-link/spec-constants/default-value/struct-with-padding.ll

```diff
@@ -0,0 +1,33 @@
+; Test checks that struct with padding is handled correctly.
+
+; RUN: sycl-post-link -split=auto -spec-const=rt -symbols -S -o %t.table %s -generate-device-image-default-spec-consts
+; RUN: cat %t.table | FileCheck %s -check-prefix=CHECK-TABLE -DPATH=%t
+; RUN: cat %t_1.prop | FileCheck %s -check-prefix=CHECK-PROP1
+; RUN: cat %t_1.ll | FileCheck %s -check-prefix=CHECK-IR1
+
+; CHECK-TABLE: [[PATH]]_0.ll|[[PATH]]_0.prop|[[PATH]]_0.sym
+; CHECK-TABLE: [[PATH]]_1.ll|[[PATH]]_1.prop|[[PATH]]_1.sym
+
+; CHECK-PROP1: specConstsReplacedWithDefault=1|1
+
+; CHECK-IR1-NOT: SpecConstant
```

Comment:
Better to use `--implicit-check-not`, because `NOT` directives are scoped, i.e. this line would allow `SpecConstant` to appear after the `store` you are looking for